README.md

```diff
@@ -366,3 +366,67 @@ $ mbedls
 ```
 
 Note how both platforms share the same platform ID (`0240`) but have a unique ending string.
+
+# Adding platform support
+
+If a platform meets the Mbed Enabled technical requirements (stated above), it can be added to Mbed LS.
+
+## Adding a new platform with a supported debugger
+
+Mbed LS currently supports the following types of debuggers:
+
+- [DAPLink](https://github.com/ARMmbed/DAPLink)
+   - As well as the related but legacy [CMSIS-DAP](https://github.com/mbedmicro/CMSIS-DAP) firmware
+- ST-LINK
+- J-Link
+
+### Adding support for DAPLink-compatible platforms (DAPLink, ST-LINK, and CMSIS-DAP)
+
+Add an entry to the `daplink` section of the [`DEFAULT_PLATFORM_DB`](https://github.com/ARMmbed/mbed-ls/blob/master/mbed_lstools/platform_database.py#L45).
+
+If your platform's name is `NEW_PLATFORM` and it has platform ID of `9999`, the new entry should be:
+
+```
+DEFAULT_PLATFORM_DB = {
+    u'daplink': {
+        ...
+        u'9999': u'NEW_PLATFORM',
+        ...
+    }
+}
+```
+
+Please order the entries by the platform ID when adding new platforms.
+
+### Adding support for J-Link platforms
+
+J-Link detection works a differently due to the information present on the platform's filesystem. All new entries should be added to the `jlink` section of the [`DEFAULT_PLATFORM_DB`](https://github.com/ARMmbed/mbed-ls/blob/master/mbed_lstools/platform_database.py#L45).
```

Comment:
"works a bit differently"

Reply:
Oops, I accidentally a word 😄 I mean to say "J-Link detection work differently".